.github/workflows/main.yml

```diff
@@ -18,10 +18,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-20.04
     steps:
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-helm-docs@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-kubepug@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-checkov@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.9.1
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-helm-docs@v1.9.2
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-kubepug@v1.9.2
+      - uses: bridgecrewio/checkov-action@v12.1708.0
```

Comment:
I see only now that the existing checkov available in pre-commit:
https://github.com/Alfresco/alfresco-build-tools/blob/master/.pre-commit-hooks.yaml#L52

was having some logic regarding which folder to pass while invoking checkov itself, along with --framework kubernetes

at least I would try adding here:
```
with:
  framework: kubernetes
```

Reply:
Done, thanks :+1: 
Do you think that would be better on the cloud-full-chart one too?

Reply:
As discussed, will add back the equivalent check in pre-commit hooks instead